cmd/init/definitionupdater/updater.go

```diff
@@ -8,12 +8,13 @@ import (
 	"github.com/gofrs/uuid"
 	"github.com/launchdarkly/go-semver"
 
+	componentstore "github.com/instill-ai/component/store"
```

Comment:
Hi @jvallesm 

Let's group the package import in this way. 
1. Golang built-in packages
2. packages from others
3. packages from others with alias
4. packages from Instill AI
5. packages from Instill AI with alias

```
"github.com/gofrs/uuid"
"github.com/launchdarkly/go-semver"

"github.com/instill-ai/pipeline-backend/pkg/datamodel"
"github.com/instill-ai/pipeline-backend/pkg/logger"
"github.com/instill-ai/pipeline-backend/pkg/repository"

componentstore "github.com/instill-ai/component/store"
errdomain "github.com/instill-ai/pipeline-backend/pkg/errors"
pb "github.com/instill-ai/protogen-go/vdp/pipeline/v1beta"
```

It's is just for a consistent convention.

Reply:
I actually prefer standard library / external / Instill (i.e. no separation between alias and no alias), many times adding an alias is just a way to avoid a conflict but in the imports the packages make sense together. However, if this is a convention already discussed, I have no problem changing it.

Reply:
Yeah, separating the alias is the convention we have discussed before.